server/container/guice/jpa-guice/src/main/java/org/apache/james/JPAJamesServerMain.java

```diff
@@ -95,10 +101,15 @@
     private static final Module JPA_MODULE_AGGREGATE = Modules.combine(JPA_SERVER_MODULE, PROTOCOLS);
 
     public static void main(String[] args) throws Exception {
+        Optional.ofNullable(System.getProperty("logback.configurationFile"))
+            .stream()
+            .forEach(file -> System.err.println("Load logging config from " + file));
```

Comment:
Can't we rely on System.out here because as far as I know we are not printing an error but an expected behavior.

Reply:
`Optional.ofNullable(...).ifPresent(...)` sounds more straightforward.

Reply:
Not sure if System.out is logged redirected/visible docker (by default). 
I will check. Thanks for the isPresent(). Will change.

Reply:
Done.